tutorials/R/README.md

```diff
@@ -306,10 +299,12 @@ The collection of trees are :
 ![alt text](./Selection_017.png "Random Forest")
 ![alt text](./Selection_018.png "Random Forest")
 
+*<b> Warning </b> : The model and predictions may vary from the example due to random sampling*
```

Comment:
would be better to use set.seed(1) so that model/predictions are the same

Reply:
ok...I'll add the seed and generate the trees again